met-api/migrations/versions/6d3c33a79c5e_add_email_verification.py

```diff
@@ -0,0 +1,41 @@
+"""Add email_verification
+
+Revision ID: 6d3c33a79c5e
+Revises: 1113e0ad66c3
+Create Date: 2022-07-28 12:57:25.200851
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '6d3c33a79c5e'
+down_revision = '1113e0ad66c3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('email_verification',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('verification_token', sa.String(length=50), nullable=False),
```

Comment:
I am kind of aligned towards calling it as token everywhere  instead of verification_token..
the verification word is duplicated

Reply:
token is so generic thought :|